usage/README.rst

```diff
@@ -50,46 +54,31 @@ or equivalently
 
    ./usage.py --help
 
-Plot data
-^^^^^^^^^
-To plot the data collected in the previous step run
-
-.. code-block:: bash
-
-    python usageplot.py -d usage.out > nodehour.out
-
-Besides making the plot, script will also return the total node-hours used by slurm on the jobs whose data is being plotted. We caught them in nodehour.out file.
-
-If you would like to know more about the arguments above, just run the following on the command line:
-
-.. code-block:: bash
-
-   python usageplot.py -h
-
-or alternately:
-
-.. code-block:: bash
-
-   python usageplot.py --help
-
 There are examples of data files and plots included in the examples subdirectory.  
 
 Advanced Plotting
 ^^^^^^^^^^^^^^^^^
-In the same directory where the data file usage.out resides, run the following in the command line:
-
-.. code-block:: bash
-
-   python usageplot.py -t w_2018_03 -d usage.out -n usage_w_2018_03 -c > nodehour.out
-
-The -t argument is the plot title (completely arbitrary), the -d argument is the name of the text file with the data (created above with usage.py), the -n argument is the name of the .png file that will be made with usageplot.py (note that you should NOT include ".png" at the end of the -n argument!!), and the -c argument will specify if you would like the plots color-coded by the Slurm jobNames that you have assigned (this will only work if singleFrameDriver.py job names start with "Co" or "Wi", mosaic.py job names start with 'mo', coaddDriver.py job names start with 'co', and multibandDriver.py job names start with 'mt').  The 'nodehour.out' file will catch the total node-hours used by slurm on the jobs whose data is being plotted.
-
-If you would prefer to not have the plots color coded, then run the follwoing in the command line:
+In the same directory where the data file usage.out resides, run the following
+in the command line:
 
 .. code-block:: bash
 
-   python usageplot.py -t w_2018_03 -d usage.out -n usage_w_2018_03 > nodehour.out
- 
+   ./usage.py -u mxk -t 'mxk node usage' -n usage_mxk -c > nodehour_elapsed.out
+
+The -u argument specifies the slurm user (or you can specify a list of jobs to
+plot with -j), the -t argument is the plot title (completely arbitrary), the -n 
+argument is the name of the .png file that will be made with usageplot.py (note
+that you should NOT include ".png" at the end of the -n argument!!), and the -c
+argument will specify if you would like the plots color-coded by the SLURM 
+jobNames that you have assigned (this will only work if singleFrameDriver.py
+job names start with "Co" or "Wi", mosaic.py job names start with 'mo',
+coaddDriver.py job names start with 'co', and multibandDriver.py job names
+start with 'mt').  The 'nodehour_elapsed.out' file will catch the total
+node-hours used by SLURM on the jobs whose data is being plotted as well as the
+elapsed run times in hours for each code run.
+
+If you would prefer to not have the plots color-coded, then don't include the
+'-c' option. 
```

Comment:
As I said some time ago, the information here is very useful but current formatting make them slightly hard to "extract". I would try to use definition list instead and describe it in more "formal" way:
```
**./usage.py** [OPTION]...

    Creates a plot illustrating **lsst-dev** cluster usage (in PNG format) based on data from SLURM accounting database. It also outputs basic statistics to standard output. 

    **-u** list
        specifies user(s) to gather data for; conflicts with **-j**

    **-j** list
        specifies job id(s) to gather data for; conflicts with **-u**

    ... and so on...
```